fvwm/conditional.c

```diff
@@ -2090,9 +2090,18 @@ void CMD_Test(F_CMD_ARGS)
 			flags_ptr = GetNextSimpleOption(flags_ptr, &var_name);
 			if (var_name)
 			{
-				const char *value = getenv(var_name);
+				const char *value;
 
-				match = (value != NULL) ? 1 : 0;
+				if ( (strlen(var_name) > 10) && (memcmp(var_name,"infostore.",10) == 0)  )
```

Comment:
I think we can drop the `strlen(var_name) > 10` check -- as in, this is a redundant  check in comparison to `memcmp()`.  This also removes a magic value (10) from the code.

Reply:
Yes boss!